### PR TITLE
[ntvcojp] fix extractor

### PR DESCRIPTION
**Pull request by nao20010128nao**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

See https://github.com/ytdl-org/youtube-dl/issues/30309

### What is the purpose of your *pull request*?
- [x] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

This fixes extractor for cu.ntv.co.jp. Geo-restricted.

closes https://github.com/ytdl-org/youtube-dl/issues/30309

### CODE REVIEW

File: yt_dlp/extractor/common.py [hunk 1]
```diff
@@ -1513,6 +1513,24 @@ def _search_nextjs_data(self, webpage, video_id, **kw):
                 webpage, 'next.js data', **kw),
             video_id, **kw)
 
+    def _search_nuxt_data(self, webpage, video_id, context_name='__NUXT__'):
```
Comment (pukkandan) [inline yt_dlp/extractor/common.py:1516]: could u make this a separate PR? (remove redundant code from sovietcloset)

File: yt_dlp/extractor/ntvcojp.py [hunk 2]
```diff
@@ -28,22 +29,31 @@ class NTVCoJpCUIE(InfoExtractor):
...
+            raise ExtractorError('Failed to extract video ID for Brightcove')
+        account_id = traverse_obj(player_config, ('player', 'account')) or '3855502814001'
+        title = traverse_obj(player_config, ('movie', 'name'))
+        if not title:
+            og_title = self._og_search_title(webpage, fatal=False) or traverse_obj(player_config, ('player', 'title'))
+            if og_title:
+                title = og_title.split('(', 1)[0].strip()
+        description = traverse_obj(player_config, ('movie', 'description'))
+        if not description:
+            description = self._html_search_meta(['description', 'og:description'], webpage)
```
Comment (pukkandan) [inline yt_dlp/extractor/ntvcojp.py:50]:
```suggestion
        description = (traverse_obj(player_config, ('movie', 'description'))
                       or self._html_search_meta(['description', 'og:description'], webpage))
```

File: yt_dlp/extractor/common.py (hunk 1 above)
Comment (nao20010128nao) [reply yt_dlp/extractor/common.py:1516]: okay

Comment (nao20010128nao) [reply yt_dlp/extractor/common.py:1516]: done #1921

**Review by pukkandan — APPROVED** [review]
This is ready once merged with master

**Comment by nao20010128nao** [comment]
done
